apps/crn-frontend/src/discover/tutorials/TutorialList.tsx

```diff
@@ -1,17 +1,34 @@
-import { FC } from 'react';
-import { DiscoverTutorialsCardList } from '@asap-hub/react-components';
-import { useDiscoverState } from '../state';
+import { TutorialsPageBody } from '@asap-hub/react-components';
+import { usePagination, usePaginationParams } from '../../hooks';
+import { useTutorials } from './state';
 
-const Tutorials: FC<Record<string, never>> = () => {
-  const { training } = useDiscoverState();
+interface TutorialListProps {
+  searchQuery: string;
+}
+
+const TutorialList: React.FC<TutorialListProps> = ({ searchQuery = '' }) => {
+  const { currentPage, pageSize } = usePaginationParams();
+  const result = useTutorials({
+    searchQuery,
+    currentPage,
+    pageSize,
+    filters: new Set(),
+  });
+
+  const { numberOfPages, renderPageHref } = usePagination(
+    result.total || 0,
```

Comment:
```suggestion
    result.total,
```
I just used this function and didn't need to provide a default. Not sure this is required